cardano-slotting/cardano-slotting.cabal

```
@@ -52,6 +52,7 @@ library testlib
   visibility:           public
   hs-source-dirs:       testlib
   exposed-modules:      Test.Cardano.Slotting.Arbitrary
+                        Test.Cardano.Slotting.Orphans
```

Let's call it `Numeric`, since `Arbitrary` and `TreeDiff` modules also provide orphan instances
```suggestion
                        Test.Cardano.Slotting.Numeric
```